### PR TITLE
Remove capstone from frida [x86_64]

### PR DESCRIPTION
**Pull request by tokatoka**

### CODE REVIEW

**Comment by tokatoka** [comment]
it's not done but, almost

File: libafl_frida/src/asan/errors.rs [hunk 1]
```diff
@@ -241,24 +243,16 @@ impl AsanErrors {
...
-                    .disasm_count(
-                        unsafe { std::slice::from_raw_parts(start_pc as *mut u8, 4 * 11) },
-                        start_pc as u64,
-                        11,
-                    )
-                    .expect("failed to disassemble instructions")
-                {
+                let insns = disas_count(
+                    &decoder,
+                    unsafe { std::slice::from_raw_parts(start_pc as *mut u8, 4 * 11) },
```
Comment (s1341) [inline libafl_frida/src/asan/errors.rs:251]: perhaps we should increase the `4` here to `16` (which is close enough to the limit of a single instruction IIRC), and then only display the first 11 instructions?

File: libafl_frida/src/asan/asan_rt.rs [hunk 2]
```diff
@@ -1245,42 +1242,42 @@ impl AsanRuntime {
...
-            108 => (Some(10), Some(64)),
-            109 => (Some(11), Some(64)),
-            110 => (Some(12), Some(64)),
-            111 => (Some(13), Some(64)),
-            112 => (Some(14), Some(64)),
-            113 => (Some(15), Some(64)),
-            41 => (Some(18), Some(64)),
+    fn register_idx(&self, reg: X86Register) -> (Option<u16>, Option<u16>) {
+        match reg {
+            X86Register::Eax => (Some(0), Some(32)),
```
[the remark this answers is not in the document]
Comment (tokatoka) [reply libafl_frida/src/asan/asan_rt.rs:1247]: ok

File: libafl_frida/src/asan/errors.rs (hunk 1 above)
Comment (tokatoka) [reply libafl_frida/src/asan/errors.rs:251]: ok

**Comment by tokatoka** [comment]
if this is resolved then good to go
https://github.com/iximeow/yaxpeax-x86/issues/31#issue-2044035648

**Pull request merged**